src/sparsezoo/model/utils.py

```diff
@@ -120,31 +139,29 @@ def load_files_from_stub(
     models = api.fetch(
         operation_body="models",
         arguments=arguments,
-        fields=["modelId", "modelOnnxSizeCompressedBytes"],
+        fields=[
+            "model_id",
+            "model_onnx_size_compressed_bytes",
+            "files",
+            "benchmark_results",
+            "training_results",
+        ],
     )
 
     if len(models):
+
         model_id = models[0]["model_id"]
```

Comment:
I'd recommend pulling out a local variable instead of repeatedly invoking `models[0]`
```python
model = models[0]
```